apps/tx-builder/src/components/Dashboard.tsx

```diff
@@ -92,74 +70,141 @@ const Dashboard = (): ReactElement => {
     return web3?.eth.ens.getAddress(name) || new Promise((resolve) => resolve(name));
   };
 
+  const isAddressInputFieldValid = isValidAddress(address) || !address;
+
   const contractHasMethods = contract && contract.methods.length > 0 && !isABILoading;
 
-  const isAddressInputFieldValid = address.length > 0 && !isValidAddress(address) ? 'The address is not valid' : '';
+  const isTransferTransaction = !abi && isValidAddress(address);
+  const isContractInteractionTransaction = abi && contractHasMethods;
+
+  const showNewTransactionForm = isTransferTransaction || isContractInteractionTransaction;
```

Comment:
Fixed an issue that showed an empty method selector when an invalid ABI was present:
![Captura de pantalla 2022-02-16 a las 12 45 27](https://user-images.githubusercontent.com/26763673/154477044-349d2ecf-601e-4585-955d-0b4ac57f361f.png)